src/Collectors/ScheduledTaskCollector.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace AG\ElasticApmLaravel\Collectors;
+
+use AG\ElasticApmLaravel\Contracts\DataCollector;
+use GuzzleHttp\Exception\ClientException;
+use Illuminate\Console\Events\ScheduledTaskFinished;
+use Illuminate\Console\Events\ScheduledTaskSkipped;
+use Illuminate\Console\Events\ScheduledTaskStarting;
+use Illuminate\Support\Facades\Log;
+use PhilKra\Events\Transaction;
+use PhilKra\Exception\Transaction\UnknownTransactionException;
+use Throwable;
+
+/**
+ * Collects info about scheduled tasks.
+ */
+class ScheduledTaskCollector extends EventDataCollector implements DataCollector
+{
+    public function getName(): string
+    {
+        return 'scheduled-task-collector';
+    }
+
+    public function registerEventListeners(): void
+    {
+        $this->app->events->listen(ScheduledTaskStarting::class, function (ScheduledTaskStarting $event) {
```

Comment:
If I'm not wrong `ScheduledTask*` events were introduced in Laravel 5.8, while we support version 5.5. Can we register the listeners only if Laravel version is `>= 5.8`?

Reply:
I've looked for it in Laravel repos and I only see those classes in 6+ 😬 . I could add a framework version check or just a class existence check (theoretically one could implement that class if they really wanted in older versions).

Reply:
Yes sorry, events for commands are the ones introduced in Laravel 5.8. We'll need checks for both collectors then.

Reply:
[`CommandStarting`](https://github.com/laravel/framework/blob/5.5/src/Illuminate/Console/Events/CommandStarting.php) and [`CommandFinished`](https://github.com/laravel/framework/blob/5.5/src/Illuminate/Console/Events/CommandFinished.php) are both in 5.5 though.